stdlib/format.mli

```diff
@@ -488,6 +488,18 @@ val safe_set_geometry : max_indent:int -> margin:int -> unit
    @since 4.08.0
 *)
 
+val pp_update_geometry : formatter -> (geometry -> geometry) -> unit
+(**
+   [pp_update_geometry ppf (fun geo -> { geo with ... })] lets you
+   update a formatter's geometry in a way that is robust to extension
+   of the [geometry] record with new fields.
+
+   Raises an invalid-argument exception if the returned geometry
```

Comment:
`Invalid_argument` instead to be more precise?

Reply:
(I want to reuse the documentation style of `pp_set_geometry`, which is similar but without the hyphen. I just removed the hyphen.)